proxy/proxies/nginx/nginx.go

```diff
@@ -50,10 +52,12 @@ import (
 )
 
 const (
-	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/ingress-nginx:v0.1.0"
+	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/nginx-ingress-controller:0.9.0-beta.8"
```

Comment:
you can specify the nginx image with cmd flag --proxy-nginx, it will be stored in image of nginx struct
See nginx.AddFlags() function